### DISCUSSION TITLE
%hostname% not working in friendlyname/devicename

### DISCUSSION THREAD

**Discussion by jeroenst** [arendst/Tasmota#17006] [Q&A] [answered]
Currently devicename/friendlyname does accept %hostname% but it doesn't change the friendlyname on the webpage.

11:00:15.247 CMD: friendlyname %hostname%
11:00:15.261 MQT: home/TASMOTA-PLUG005;/stat/RESULT = {"FriendlyName1":"%hostname%"}
11:00:19.508 CMD: friendlyname
11:00:19.525 MQT: home/TASMOTA-PLUG005;/stat/RESULT = {"FriendlyName1":"%hostname%","FriendlyName2":"Tasmota2","FriendlyName3":"Tasmota3","FriendlyName4":"Tasmota4","FriendlyName5":"","FriendlyName6":"","FriendlyName7":"","FriendlyName8":""}
11:00:32.094 MQT: home/TASMOTA-PLUG005;/stat/STATUS11 = {"StatusSTS":{"Time":"2022-11-06T11:00:32","Uptime":"0T00:04:06","UptimeSec":246,"Heap":19,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"POWER":"OFF","Wifi":{"AP":1,"SSId":"J&M IOT","BSSId":"06:EC:DA:8D:48:02","Channel":5,"Mode":"11n","RSSI":88,"Signal":-56,"LinkCount":1,"Downtime":"0T00:00:07"}}} (retained)
11:00:50.777 CMD: hostname
11:00:50.793 MQT: home/TASMOTA-PLUG005;/stat/RESULT = {"Hostname":"TASMOTA_PLUG005"}

Is it possible to change Tasmota so %hostname% can be used for devicename/friendlyname?

**Comment by sfromis** [comment, marked as answer]
Your suggestion is completely contrary to the what FriendlyName is meant for, to be useful for non-techies, and in voice assistants.

**Reply by jeroenst** [reply]
Nevermind, SetOption53 works for me.

And I meant devicename instead of friendlyname.
